src/com/mateakademia/homework/homework6/Chamomile.java

```diff
@@ -0,0 +1,13 @@
+package com.mateakademia.homework6;
+public class Chamomile extends Flowers {
+
+    public Chamomile(){
+        super(70);
```

Comment:
better pass value as constructor param or use constants instead of using "magic numbers"